packages/cookie-consent/cookie-consent.scss

```diff
@@ -0,0 +1,88 @@
+@import "~@fremtind/jkl-core/variables/all";
+@import "~@fremtind/jkl-core/mixins/all";
+@import "~@fremtind/jkl-button/button.scss";
+@import "~@fremtind/jkl-list/list.scss";
+
+$card-corner-radius: rem(2px);
+$container-width: rem(660px);
+$container-width--compact: rem(320px);
+
+.jkl-cookie-consent-modal + backdrop {
+    position: fixed;
+    background: rgba(0, 0, 0, 0.3);
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    transition: 0.2s;
+    z-index: $z-index--modal !important;
+}
+.jkl-cookie-consent-modal[hidden],
+.jkl-cookie-consent-modal + backdrop[hidden] {
+    pointer-events: none;
+    visibility: hidden;
+    display: block;
+    opacity: 0;
+}
+
+.jkl-cookie-consent-modal {
+    z-index: $z-index--floating !important;
+    position: absolute;
+    margin: auto;
+    top: 15%;
+    left: 0;
+    right: 0;
+    width: 100%;
+    padding: $layout-spacing--medium $layout-spacing--medium rem(30px) $layout-spacing--medium;
+    min-width: $container-width--compact;
+    max-width: $container-width;
+    background: $hvit;
+    box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+    transition: 0.2s;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+
+    &.jkl-cookie-consent-modal--settings {
+        ul.jkl-list {
+            margin-bottom: $component-spacing--large;
+        }
+    }
+
+    @include jkl-text-style("desktop/body");
+
+    &__header {
+        @include jkl-text-style("desktop/heading-medium");
+        margin-bottom: $layout-spacing--small;
+    }
+
+    ul.jkl-list {
+        margin: 0;
+
+        .jkl-list__item {
+            margin-top: 0;
+
+            &:not(:last-child) {
+                margin-bottom: $component-spacing--large;
+            }
+        }
+    }
+
+    p,
+    fieldset {
+        @include jkl-text-style("desktop/small");
+    }
+
+    .jkl-checkbox {
+        margin-bottom: $component-spacing--small;
+        margin-top: $component-spacing--xl;
+    }
+
+    &__button-group {
+        margin-top: $layout-spacing--medium;
+
+        .jkl-button:not(:first-of-type) {
+            margin-left: $layout-spacing--small;
+        }
```

Comment:
Hakk i plata her, så du skjønner sikkert hva jeg skal kommentere 🙃 Her hadde det vel uansett holdt med bare `:not(:first-child)`?

Reply:
Endret litt på syntaxen til denne så den ikke peker på `jkl-button` i det hele tatt. Enig i forenklingen til `:first-child` 👍